paddle/fluid/operators/reshape_op.cc

```diff
@@ -553,9 +560,12 @@ REGISTER_OPERATOR(reshape_grad, ops::ReshapeGradOp,
 
 REGISTER_OP_CPU_KERNEL_FUNCTOR(reshape, float, ops::ReshapeKernel, double,
                                ops::ReshapeKernel, int, ops::ReshapeKernel,
-                               int64_t, ops::ReshapeKernel);
+                               int8_t, ops::ReshapeKernel, uint8_t,
```

Comment:
I think we do not need the int8 kernels for the `reshape` op. We are interested in quantizing `reshape2` only now.